app/models/scheduled_mailing.rb

```diff
@@ -18,14 +18,14 @@ def deliver!
   end
 
   def cancel!
-    update!(cancelled_at: Time.now)
+    update!(cancelled_at: Time.current)
```

Comment:
Just to check, because this is now used as part of process, bearing in mind
```
irb(main):020:0> Time.now
=> 2021-01-18 12:36:48.667092 +0000
irb(main):021:0> Time.current
=> Mon, 18 Jan 2021 12:36:50 GMT +00:00
```
the latter includes a more prettified date this shouldn't affect anything right as it's still a Time object correct? Asking in case it needs to be in a specified format but I'm guessing look at what was prior probably not

Reply:
postgres saves it as UTC so it should be ok.

Also, since we set timezone to europe/london then https://api.rubyonrails.org/classes/ActiveSupport/TimeZone.html
will do the clever math of converting UTC to the relevant timezone